packages/nimble-components/src/utilities/tests/matrix.ts

```diff
@@ -21,7 +21,7 @@ export const sharedMatrixParameters = () => ({
             hidden: true
         }
     }
-} as const);
+}) as const;
```

Comment:
Can we just remove the outer `()`?

Reply:
Same for `storybook.ts`

Reply:
Nope, you can't, it's an arrow function returning an object, have to wrap as an expression
Following is ambiguous:
`() => {test:2}`
Make it an expression:
`() => ({test:2})`